### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by qsdstefan**
Updated README.md with package info, requirements, installation, setup and example code sections.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,192 @@
...
+});
+```
+
+### Using Laravel Echo on client-side
+
+Laravel Echo is a JavaScript library that makes it painless to subscribe to channels and listen for events broadcast by your server-side broadcasting driver.
+
+1. Install Laravel Echo and Ably:
+```
+npm install --save-dev laravel-echo ably
```
Comment (owenpearson) [inline README.md:91]: I assume this won't actually work with the official `laravel-echo` NPM module - should this be `@ably/laravel-echo`?

Comment (owenpearson) [reply README.md:91]: And we should also add something explaining that we maintain a fork of the official laravel-echo module which allows you to use the official ably-js SDK.

File: README.md [hunk 2]
```diff
@@ -1 +1,192 @@
...
+
+## Example code
+
+### Registering channels
+
+You can define channel capabilities for private and presence channels in `routes/channels.php`.
+
+**Private chanel**
+
+For private channels, access is allowed for truthy values and denied for falsy values.
```
Comment (owenpearson) [inline README.md:69]: I think we should explain that if the response is truthy it should be an ably capability object and link to the ably docs for capabilities

File: README.md [hunk 3]
```diff
@@ -1 +1,192 @@
...
+```php
+Broadcast::channel('channel1', function ($user) {
+    return ['capability' => ["subscribe", "history"]];
+});
+```
+
+**Presence channel**
+
+For presence channels, we can also return data about the user. ([read more](https://laravel.com/docs/9.x/broadcasting#authorizing-presence-channels))
```
Comment (owenpearson) [inline README.md:78]:
```suggestion
For presence channels, you can also return data about the user. ([read more](https://laravel.com/docs/9.x/broadcasting#authorizing-presence-channels))
```

**Review by owenpearson — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (qsdstefan) [reply README.md:69]: Added a new sentence, please check

**Review by qsdstefan — COMMENTED** [review]
Made changes as per @owenpearson's suggestions

**Review by owenpearson — APPROVED** [review]

**Review by sacOO7 — APPROVED** [review]
LGTM : )

**Pull request merged**